api/app/scripts/setup_readonly_sql_query.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Directory where SQL scripts are stored
+SQL_DIR="./scripts/readonly_user_setup"
+
+# Export the database password for psql command
+export PGPASSWORD="$DATABASE_PASSWORD"
+
+# Check if directory exists
+if [ ! -d "$SQL_DIR" ]; then
+    echo "Directory $SQL_DIR does not exist."
+    exit 1
+fi
+
+# Find all SQL files in the directory, sort them numerically, and loop through each one
+for sql_file in $(ls $SQL_DIR/*.sql | sort -V); do
+    echo "Running $sql_file ..."
+
+    # Create a temporary file for the modified SQL
+    temp_file=$(mktemp)
+
+    # Replace the placeholder string with the real password
+    sed "s/string_to_replace_with_real_password/$READONLY_SQL_DATABASE_PASSWORD/g" "$sql_file" > "$temp_file"
```

Comment:
Worth checking `READONLY_SQL_DATABASE_PASSWORD ` isn't empty.

Reply:
Yes I can do. This code isn't run against prod mind you so it's not the worst thing in the world if it were empty but yeah I'll change it